src/api/app/controllers/webui/package_controller.rb

```diff
@@ -921,10 +921,10 @@ def buildresult
       show_all = params[:show_all] == 'true'
       @index = params[:index]
       @buildresults = @package.buildresult(@project, show_all)
-      switch_to_webui2
+      switch_to_webui2 if params[:switch].present?
```

Comment:
Phew....

Reply:
shouldn't we have this generically for all webui2 routes as debugging aid?

Reply:
How do you mean?

Reply:
I think it's generally useful for a developer in beta group to look up a route in old webui to check if the bug is specific to webui2.

Reply:
Just disable the beta program? :-)